GetStream/stream-chat-swift
# Add support for shadow banning a member

### 🔗 Issue Links
Resolves https://github.com/GetStream/ios-issues-tracking/issues/424

### 🎯 Goal
Add support for shadow banning a member and exposing `ChatChannelMember.isShadowBannedFromChannel`.

### 📝 Summary
- Added `ChatMemberController.shadowBan()`
- Added handling for Shadow Banning events
- Exposed `ChatChannelMember.isShadowBannedFromChannel`

### 🧪 Manual Testing Notes
- Open Channel List
- Swipe a channel
- Tap on Shadow ban a member
- Select the member to shadow ban
- Swipe a channel
- Tap on "Show Banned Members"
- The member should appear on the list
- If that member sends a message, it should display as shadowed message

### ☑️ Contributor Checklist
